### PR TITLE
UI Improvements

### PR DESCRIPTION
**Pull request by MeijerM1**
This PR implements the new designs based on the [InVision designs](https://projects.invisionapp.com/d/main?origin=v7#/projects/prototypes/20080254?scrollOffset=218).

Since running the application locally can be a bit of hassle (you need to run the backend to actually do anything). The plan is to use this PR to review the code. When it is on staging we will do an actual review of the UI itself.

### CODE REVIEW

File: src/components/Circle/Circle.tsx [hunk 1]
```diff
@@ -15,16 +15,20 @@ function CustomCircle(props: Props): React.ReactElement {
     <div className={s.wrapper}>
       {props.currentKudos != null && (
         <span className={s.title} style={{ color: 'grey' }}>
-          <h2 data-testid="current-kudos" style={{ color: 'rgba(0, 0, 0, 0.85)' }}>{props.currentKudos}₭</h2>
-          <span data-testid="goal-kudos">of {props.neededKudos}₭ for {props.goal}</span>
+          <h2 data-testid="current-kudos" className={s.current}>
+            {props.currentKudos}
+            <span style={{ fontSize: '35px', margin: 0, padding: 0 }}>₭</span>
```
Comment (RoelN) [inline src/components/Circle/Circle.tsx:20]: Why inline styles here?

File: src/components/navigation/Desktop.module.scss [hunk 2]
```diff
@@ -2,8 +2,12 @@
   position: fixed;
   top: 0;
   width: 100%;
+  background: #00749A;
```
Comment (RoelN) [inline src/components/navigation/Desktop.module.scss:5]: Nit: lowercase

File: src/components/navigation/Desktop.module.scss [hunk 3]
```diff
@@ -2,8 +2,12 @@
...
 }
 
-.menu {
-  height: 62.84px;
-}
+:global(#root) {
+  .menu {
+    height: 62.84px;
+    background: #00749A;
```
Comment (RoelN) [inline src/components/navigation/Desktop.module.scss:11]: Nit: lowercase (also, ugh that magic number!

File: src/components/navigation/Desktop.tsx [hunk 4]
```diff
@@ -34,9 +34,14 @@ export interface Props {
 export function DesktopNavigation(props: Props) {
   return (
     <div className={s.root}>
-      <Menu fixed="top" inverted size="large" className={s.menu}>
+      <Menu inverted fixed="top" size="large" className={s.menu}>
         <Container>
-          <Menu.Item data-testid="home-button" onClick={() => props.history.push(PATH_FEED)}>Home</Menu.Item>
+          <Menu.Item
+            style={{ borderLeft: 'none' }}
```
Comment (RoelN) [inline src/components/navigation/Desktop.tsx:40]: Why inline?

File: src/components/navigation/Toolbar.module.scss [hunk 5]
```diff
@@ -15,10 +20,15 @@
 
 .top_navigation {
   width: 100%;
-  position: absolute;
+  position: static;
   height: 68px;
-  background-color: white;
+  background-color: #00749A;
```
Comment (RoelN) [inline src/components/navigation/Toolbar.module.scss:25]: Nit: lowercase

File: src/components/navigation/Toolbar.module.scss [hunk 6]
```diff
@@ -15,10 +20,15 @@
...
   z-index: 999;
   display: flex;
   flex-direction: row;
+
+  span {
+    color: white;
```
Comment (RoelN) [inline src/components/navigation/Toolbar.module.scss:31]: Any reason to use a mix of `white` and `#ffffff`?

File: src/modules/choose-team/CreateTeamPage.tsx [hunk 7]
```diff
@@ -92,56 +93,70 @@ class CreateTeamPage extends Component<Props, State> {
...
-                    </Message>
-                  )}
-                </Form>
-              )}
-            </Mutation>
-          </div>
-        </div>
+        <Responsive minWidth={Responsive.onlyTablet.minWidth}>
+          <Segment
+            style={{
```
Comment (RoelN) [inline src/modules/choose-team/CreateTeamPage.tsx:147]: Why inline?

File: src/modules/choose-team/components/ChooseTeam.module.scss [hunk 8]
```diff
@@ -1,3 +1,3 @@
 .button {
-  margin: 0 !important;
+  margin: 1rem;
```
Comment (RoelN) [inline src/modules/choose-team/components/ChooseTeam.module.scss:2]: Nice, good riddance of that `!important` 👍

File: src/modules/feed/FeedPage.module.scss [hunk 9]
```diff
@@ -1,22 +1,73 @@
...
+      margin: auto;
+    }
+  }
+
+  .board_header {
+    color: #FFFFFF;
+    font-family: Rajdhani, sans-serif;
+    font-weight: lighter;
+    font-size: 34px;
+    letter-spacing: 0;
```
Comment (RoelN) [inline src/modules/feed/FeedPage.module.scss:61]: This explicitly disables justification of text. Maybe you want `normal` here?

File: src/modules/manage-team/ManageTeamPage.module.scss [hunk 10]
```diff
@@ -0,0 +1,21 @@
...
+    padding: 36px 0;
+
+    div {
+      border: none;
+    }
+
+    a {
+      width: 100%;
+      border: none;
+      border-radius: 0 !important;
```
Comment (RoelN) [inline src/modules/manage-team/ManageTeamPage.module.scss:12]: No way around using `!important`?

File: src/modules/manage-team/ManageTeamPage.module.scss [hunk 11]
```diff
@@ -0,0 +1,21 @@
...
+      font-weight: 700;
+    }
+  }
+
+  .column {
+    padding-top: 0 !important;
```
Comment (RoelN) [inline src/modules/manage-team/ManageTeamPage.module.scss:18]: No way around using `!important`?

File: src/modules/statistics/Statistics.module.scss [hunk 12]
```diff
@@ -13,3 +26,19 @@
...
+  margin: 0;
+  font-size: 36px;
+  font-weight: 500;
+  letter-spacing: 0;
+  line-height: 45px;
+  text-align: center;
+}
+
+.current_percentage_text {
+  float: right;
```
Comment (RoelN) [inline src/modules/statistics/Statistics.module.scss:41]: Maybe use `flex` to align, as with other elements in the app?

File: src/modules/statistics/Statistics.tsx [hunk 13]
```diff
@@ -41,19 +43,12 @@ export interface ActiveGoal {
...
-        color: 'grey',
-        display: 'relative',
-      }}
-    >
-      ₭udometer
-    </h2>
-    <h4 style={{ marginTop: '0.6rem' }}>{moment().format('MMMM Do, YYYY')}</h4>
+  <div className={s.container}>
+    <h2 className={s.kudo_header}>₭udometer</h2>
+    <h4 style={{ color: '#FFF', marginTop: '0.6rem' }}>{moment().format('MMMM Do, YYYY')}</h4>
```
Comment (RoelN) [inline src/modules/statistics/Statistics.tsx:51]: Looks like the hierarchy is off here? h2, h4, then h3?

**Comment by MeijerM1** [comment]
@RoelN I removed all the inline styles, extracted all color usage to css vars (I build a very, very simple theme file) and I removed all the importants where possible, in some places Semantic UI actually uses it so I need to use it as well to override it. 

I'd like to get rid of the magic numbers used in all the media queries but because of React CSS Modules I can't have a single file that defines all these variables and import that before all the other styles. 

One approach would be to define a single file with only variables in it and import that file in every `.scss` file, but that seems like a code smell to me.

File: src/modules/feed/FeedPage.module.scss [hunk 14]
```diff
@@ -1,22 +1,95 @@
...
+
+  .form {
+    margin: auto;
+    width: 100%;
+    max-width: 512px;
+  }
+
+  .board_container {
+    max-width: 480px;
+    -webkit-overflow-scrolling: touch;
```
Comment (RoelN) [inline src/modules/feed/FeedPage.module.scss:41]: Small nit: if you use prefixed properties, leave a quick comment why this is only needed for `-webkit`

**Comment by RoelN** [comment]
@MeijerM1 Re: variables, maybe you can use a global stylesheet that isn't scoped to a component, and declare them there?

Otherwise, nice job, and thanks for cleaning up!

**Pull request merged**